### PR TITLE
Fix the bug of insert/delete/update

### PR DESCRIPTION
**Pull request by nnsgmsone**
## What type of PR is this?

- [ ] API-change
- [X] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:

### CODE REVIEW

**Review by fengttt — COMMENTED** [review]
What is Zs?   Why do people missed this thing, and how can we help people not make this kind mistake?

File: pkg/sql/colexec/insert/insert.go [hunk 1]
```diff
@@ -96,6 +97,10 @@ func Call(_ int, proc *process.Process, arg any) (bool, error) {
 
 		}
 	}
+	// XXX The original logic was buggy and I had to temporarily circumvent it
```
Comment (fengttt) [inline pkg/sql/colexec/insert/insert.go:100]: Well ...   Why was buggy?   What is Zs?   Why does this one help?

Hack it fine, but please explain.

**Comment by nnsgmsone** [comment]
> What is Zs? Why do people missed this thing, and how can we help people not make this kind mistake?

The number of occurrences of the tuple represented by zs(I keep this for ivm) and how to help others avoid this. Let me think about it. I'll figure out how to make users not have to care about this.

**Review by ouyuanning — APPROVED** [review]

File: pkg/sql/colexec/insert/insert.go (hunk 1 above)
Comment (nnsgmsone) [reply pkg/sql/colexec/insert/insert.go:100]: ok，the upper-level code may not be populated with zs. As for zs, I will follow up to figure out how to deal with it.

**Pull request merged**